extra/mariabackup/backup_mysql.cc

```diff
@@ -1395,7 +1397,8 @@ write_xtrabackup_info(MYSQL *connection, const char * filename, bool history)
 		|| xtrabackup_databases_exclude
 		);
 
-	backup_file_printf(filename,
+	char *buf = NULL;
+	int buf_len = asprintf(&buf,
```

Comment:
It's worth checking for `buf_len < 0` here, `backup_file_printf` had some builtin error checks.